tools/serve/serve.py

```diff
@@ -845,6 +846,61 @@ def start_quic_transport_server(logger, host, port, paths, routes, bind_address,
         startup_failed(logger)
 
 
+class WebTransportH3Daemon(object):
+    def __init__(self, host, port, handlers_path=None, private_key=None, certificate=None, log_level=None):
+        args = ["python3", "wpt", "serve-webtransport-h3"]
+        if host:
+            args += ["--host", host]
+        if port:
+            args += ["--port", str(port)]
+        if private_key:
+            args += ["--private-key", private_key]
+        if certificate:
+            args += ["--certificate", certificate]
+        if handlers_path:
+            args += ["--handlers-path", handlers_path]
+        if log_level == "debug":
+            args += ["--verbose"]
+        self.command = args
+        self.proc = None
+
+    def start(self):
+        def handle_signal(*_):
+            if self.proc:
+                try:
+                    self.proc.terminate()
+                except OSError:
+                    # It's fine if the child already exits.
+                    pass
+                self.proc.wait()
+            sys.exit(0)
+
+        signal.signal(signal.SIGTERM, handle_signal)
+        signal.signal(signal.SIGINT, handle_signal)
+
+        self.proc = subprocess.Popen(self.command)
+        # Give the server a second to start and then check.
+        time.sleep(1)
```

Comment:
Removed (and added a sleep in a different place to work around lack of health check for UDP ports).

Reply:
This line of code is still here, did you forget to push some change?

Reply:
I needed to re-add this since the server now runs in a separate process. Heal check for UDP ports in a different place didn't work well (being a flaky).